OrbitQt/CallTreeWidget.h

```diff
@@ -46,6 +46,65 @@ class CallTreeWidget : public QWidget {
   static const QString kActionDisassembly;
   static const QString kActionCopySelection;
 
+  template <
+      typename HideValuesProxyModelT,
+      typename = std::enable_if_t<std::is_base_of_v<QIdentityProxyModel, HideValuesProxyModelT>>>
```

Comment:
If I'm not mistaken you want the compilation to fail if `HideValuesProxyModelT` does not subclass from QIdentityProxyModel, right?

If yes you probably don't need SFINAE here. A `static_assert` inside the function would be much nicer since you can also specify an error message.

You only need SFINAE if you wanted to remove this function from an overload set, meaning there were another `SetCallTreeView(...)` overload which should be picked instead of this one.

Reply:
Good point. Either way I removed the type parameter alrogether as per the other comment.